.rubocop.yml

```diff
@@ -1,9 +1,13 @@
 AllCops:
+  TargetRubyVersion: 2.7
```

Comment:
is this mean the service has to run 2.7.x ruby to use this lograge? is it not going to stop us from upgrade the ruby version on the service repos?